inc/composer/class-command.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace Altis\Dev_Tools\Composer;
+
+use Composer\Command\BaseCommand;
+use Symfony\Component\Console\Exception\CommandNotFoundException;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Chassis command for Composer.
+ */
+class Command extends BaseCommand {
+	/**
+	 * Configure the command.
+	 */
+	protected function configure() {
+		$this->setName( 'dev-tools' );
+		$this->setDescription( 'Developer tools' );
+		$this->setDefinition( [
+			new InputArgument( 'subcommand', InputArgument::REQUIRED, 'phpunit, scaffold' ),
+			new InputOption( 'chassis', null, null, 'Run commands in the Local Chassis environment' ),
+			new InputArgument( 'options', InputArgument::IS_ARRAY ),
+		] );
+		$this->setHelp(
+			<<<EOT
+Run a dev tools feature.
+
+To run PHPUnit integration tests:
+    phpunit [--chassis] [--] [options]
+                                use `--` to separate arguments you want to
+                                pass to phpunit. Use the --chassis option
+                                if you are running Local Chassis.
+Create scaffolding for various tools:
+    scaffold <type>             possible values are 'phpunit' and 'travis'
```

Comment:
I think we should look to have no scaffolding, and zero-config, at least as a starting point, where we can allow specific configuration changes in the future as they are needed. This boilerplate approach I think will lead us to a similar place as hm-base where we have a lot of copies of the scaffold at varying versions / revisions.